packages/tooltip/Tooltip.test.tsx

```diff
@@ -1,18 +1,62 @@
-import { Tooltip } from './Tooltip';
-
+import { Tooltip } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
+import {WrappedNode} from "bore";
 
 describe( Tooltip.is, () => {
 
-  it( 'should render', () => {
+  describe( 'JSX', () => {
+
+    it( 'should render', () => {
+
+      return mount(
+        <Tooltip label="hello"/>
+      ).wait(expectDefaultRender);
 
-    console.warn('Missing tests for Tooltip!');
+    });
 
-    return mount(
-      <Tooltip label="hello"/>
-    ).wait();
+    it( 'should render with top tooltip', () => {
+
+      return mount(
+        <Tooltip label="hello" type="top" />
+      ).wait(expectRenderWithTopType);
+
+    });
 
   });
 
+  describe( 'Element', () => {
+
+    it( 'should render', () => {
+
+      return mount(
+        <bl-tooltip label="hello"></bl-tooltip>
+      ).wait(expectDefaultRender);
+
+    });
+
+    it( 'should render with top tooltip', () => {
+
+      return mount(
+        <bl-tooltip label="hello" type="top"></bl-tooltip>
```

Comment:
just a reminder how JSX works ( with bore `h` ):
` <Tooltip label="hello" type="top" />` transforms to:
- `h( Tooltip, {label:'hello', type:'top'} )` 
- which is created by `new Tooltip()`

`<bl-tooltip label="hello" type="top"></bl-tooltip>` transforms to:
- `h('bl-tooltip',{label:'hello', type:'top'})`
- which is created by `document.createElement`